src/pathwayvis/components/map/map.component.ts

```diff
@@ -496,6 +516,19 @@ class MapComponentCtrl {
     this.$scope.$apply(() =>
       this.processActionClick(this.contextActions[1], this.contextElement));
   }
+
+  public handleChangeBounds(args) {
+    const bounds = {lower: 0, upper: 0};
+    const tooltipContainer = d3.select('div#tooltip-container');
+    bounds.lower = parseInt(tooltipContainer.select('#lowerbound').property("value"));
+    bounds.upper = parseInt(tooltipContainer.select('#upperbound').property("value"));
+    console.log('[change-bounds]', bounds);
+    this.contextElement = Object.assign({}, this.contextElement, {
```

Comment:
btw in most cases `Object.assign` is not needed. You can do
`this.contextElement = {...this.contextElement, bounds};`